From 4165c5f6791d7f489ca423cfafe5ab5b950c323d Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Wed, 19 Oct 2016 00:35:36 +0200
Subject: [PATCH] Permit freeciv-manual in client less builds.

Freeciv-web uses freeciv-manual to get some help texts Freeciv generates
from the ruleset. Having to compile the C clients would slow down the
Freeciv-web build. The build system is capable of compiling the client
source files freeciv-manual needs.

In an earlier "freeciv-manual without the client" experiment a difference in
the generated server commands and server setting output was observed. This
was interpreted as corruption caused the client not being compiled. That
experiment couldn't be replicated. The difference may have been caused by an
accidental variation in applied patches and compile time settings.

See patch #7874
---
 tools/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/Makefile.am b/tools/Makefile.am
index 6f939fc..1a41d11 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -13,12 +13,10 @@ include $(top_srcdir)/bootstrap/Makerules.mk
 bin_PROGRAMS = freeciv-ruleup
 
 if SERVER
-if CLIENT
 if FCMANUAL
 bin_PROGRAMS += freeciv-manual
 endif
 endif
-endif
 
 if MODINST_CLI
 bin_PROGRAMS += freeciv-mp-cli
-- 
2.1.4

